### ISSUE TITLE
Possible false positive for angular/di

### ISSUE THREAD

**Issue by faceleg** [Gillespie59/eslint-plugin-angular#265]
The offending code

```JS
(function() {
  'use strict';
  // 4:3  error  You should use the function syntax for DI  angular/di
  angular.module('app')
    .config('$compileProvider', compileConfig);

  function compileConfig($compileProvider) {
    $compileProvider.debugInfoEnabled(false);
  }
})();
```

**Comment by faceleg** [comment]
I'm a moron.

```JS
(function() {
  'use strict';
  // 4:3  error  You should use the function syntax for DI  angular/di
  angular.module('app')
    .config(compileConfig);

  compileConfig.$inject = ['$compileProvider'];

  function compileConfig($compileProvider) {
    $compileProvider.debugInfoEnabled(false);
  }
})();
```
